crates/router/src/connector/riskified/transformers/api.rs

```diff
@@ -163,7 +163,7 @@ impl TryFrom<&frm_types::FrmCheckoutRouterData> for RiskifiedPaymentsCheckoutReq
                     .get_order_details()?
                     .iter()
                     .map(|order_detail| LineItem {
-                        price: order_detail.amount,
+                        price: order_detail.amount.get_amount_as_i64(), // This should be changed to MiniorUnit when we implement amount conversion for this connector. Additionally, the function get_amount_as_i64() should be avoided in the future.
```

Comment:
Nit: Typo.
Update this in the other places you've added the comment as well.

```suggestion
                        price: order_detail.amount.get_amount_as_i64(), // This should be changed to MinorUnit when we implement amount conversion for this connector. Additionally, the function get_amount_as_i64() should be avoided in the future.
```